internal/core/services/identity.go

```diff
@@ -355,16 +358,30 @@ func (i *identity) UpdateState(ctx context.Context, did w3c.DID) (*domain.Identi
 				return fmt.Errorf("error saving new identity state: %w", err)
 			}
 
-			err = i.rhsPublisher.PushHashesToRHS(ctx, newState, previousState, updatedRevocations, iTrees)
+			rhsPublishers, err := i.rhsFactory.BuildPublishers(ctx, reverse_hash.RHSMode(i.credentialStatusSettings.RHSMode), &kms.KeyID{
+				Type: kms.KeyTypeEthereum,
+				ID:   i.credentialStatusSettings.OnchainTreeStore.PublishingKeyPath,
+			})
 			if err != nil {
-				log.Error(ctx, "publishing hashes to RHS", "err", err)
-				if i.ignoreRHSErrors {
-					err = nil
-				} else {
-					return err
-				}
+				log.Error(ctx, "can't create RHS publisher", "err", err)
+				return fmt.Errorf("can't create RHS publisher: %w", err)
 			}
 
+			if len(rhsPublishers) >= 0 {
```

Comment:
This if is unneded. len of something is always >=0. 

Or maybe it should be if >0 ?